src/fabric/src/fabric2_db.erl

```diff
@@ -359,17 +374,17 @@ get_pid(#{}) ->
 
 
 get_revs_limit(#{} = Db) ->
-    RevsLimitBin = fabric2_fdb:transactional(Db, fun(TxDb) ->
-        fabric2_fdb:get_config(TxDb, <<"revs_limit">>)
+    #{revs_limit := RevsLimit} = fabric2_fdb:transactional(Db, fun(TxDb) ->
+        fabric2_fdb:ensure_current(TxDb)
     end),
-    ?bin2uint(RevsLimitBin).
+    RevsLimit.
 
 
 get_security(#{} = Db) ->
-    SecBin = fabric2_fdb:transactional(Db, fun(TxDb) ->
-        fabric2_fdb:get_config(TxDb, <<"security_doc">>)
+    #{security_doc := SecDoc} = fabric2_fdb:transactional(Db, fun(TxDb) ->
+        fabric2_fdb:ensure_current(no_security_check(TxDb))
```

Comment:
Why are you setting no_security_check here explicitly but not in get_revs_limit?

Reply:
The idea is that we'd need to be able to read the security doc to check membership but for get_revs_limit we don't need to do it

Reply:
Gotchya.